src/components/SignUp/index.css

```diff
@@ -0,0 +1,43 @@
+:root {
+  --title-color: #002736;
+  --subTitle-color: #000000;
+  --background-color: #fc9e01;
+  --text-color: #eff4f6;
+  --filter-shadow-color: #b3b3b3;
+  --link-color: #0641af;
+}
+
+.signupContainer {
+  display: flex;
```

Comment:
we can use the bootstrap class instead of display flex

Reply:
edited.